config.yaml

```diff
@@ -1,6 +1,6 @@
 baseURL: http://numpy.org/
 languageCode: en-us
-title: Numpy.org
+title: Numpy
```

Comment:
NumPy?

Reply:
Should this be NumPy? I can change this.

Reply:
yes, it's NumPy